TAs/optee_ta/AuthVars/lib/sub.mk

```
@@ -1,7 +1,13 @@
 .PHONY: create_lib_symlinks
-create_lib_symlinks: ./lib/wolf/wolf_symlink
+create_lib_symlinks:
 
 .PHONY: clean_lib_symlinks
-clean_lib_symlinks: remove_wolf_symlink
+clean_lib_symlinks:
 
-subdirs-y += wolf
\ No newline at end of file
+ifeq ($(CFG_FTPM_USE_WOLF),y)
```

AuthVars should not refer to configuration variables of another TA (even in name only)
```suggestion
ifeq ($(CFG_AUTHVARS_USE_WOLF),y)
```

---

Good catch, thanks.